docs/python.org/CVE-Numbering-Authority.md

```diff
@@ -0,0 +1,151 @@
+---
+title: CVE Numbering Authority
+---
+# CVE Numbering Authority
+
+## Points of Contact (POC)
+
+- Seth Michael Larson (`seth@python.org`)
```

Comment:
I think just names are fine for this doc. Might summarize this by including "All Points of Contact are subscribed to [the CNA email list](mailto:cna@python.org)"

Reply:
This applies in later sections that have email addresses as well.

Reply:
Done!